iracema/plot.py

```diff
@@ -11,6 +11,34 @@
 
 import iracema.features as features
 
+def plot(audio,
```

Comment:
In the context of this module, the name `plot` is not specific enough (after all, all methods in this modules are for plotting). I recommend you to name it `plot_curve`.